components/tag-list/demo/tag-list.html

```diff
@@ -82,7 +82,7 @@ <h2>Clearable Tag List</h2>
 							e.target.removeChild(tag);
 						});
 						// consumer must move focus someplace logical
-						forceFocusVisible(nextFocusable);
+						if (nextFocusable) nextFocusable.focus();
```

Comment:
Could just do `nextFocusable?.focus()`? Or are we hesitant to use fancy techniques in demos lest they get applied with bad `.browerslist`s? 😅